build/yamls/antrea-gke-node-init.yml

```diff
@@ -0,0 +1,51 @@
+kind: DaemonSet
```

Comment:
I think @antoninbas suggested to put it in: build/yamls/gke

Reply:
Lets keep it outside. We have antrea-gke, antrea-eks outside. There is no folder for it. If dir is preferred, we should do it for eks too.

Reply:
It's fine as it is, now that "-gke-" is in the file name